Makefile

```diff
@@ -6,4 +6,11 @@ lint:
 	flake8 stacker_blueprints
 
 test:
-	python setup.py test
+	python setup.py nosetests \
```

Comment:
Can we pull these coverage changes into a separate pull-request?

Reply:
Yup.